python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -701,6 +775,18 @@ class carrying all of the information for the object whose image
                                   add_to_image=True,
                                   gain=detector.photParams.gain)
 
+                    # If we are writing centroid files,store the entry.
+                    if self.centroid_base_name is not None:
+                        centroid_tuple = (detector.fileName, bandpassName, gsObject.uniqueId,
+                                          gsObject.flux(bandpassName), xPix, yPix)
+                        self.centroid_list.append(centroid_tuple)
+
+                else:
```

Comment:
This `else` block was removed in #52 [`here`](https://github.com/lsst/sims_GalSimInterface/pull/52/files#diff-16d65580d1df4bdb22bfac6e8ba298f0L691).  Not sure why it's showing up in this PR since it was rebased.  Maybe the added newline before the `else:`?  In any case, this `else` block isn't relevant any longer.